cmd/pr_status/pr_status.go

```diff
@@ -0,0 +1,150 @@
+	"THUMBS_UP":   "👍",
+	"THUMBS_DOWN": "👎",
+	"LAUGH":       "😆",
+	"HOORAY":      "🎉",
+	"CONFUSED":    "😕",
+	"HEART":       "❤️",
+	"ROCKET":      "🚀",
+	"EYES":        "👀",
+}
+
+var gh github.GitHub = github.NewRealGitHub()
+
+var list bool
+
+func NewPrStatusCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "pr-status",
+		Short: "Displays the status of PRs",
+		Run:   run,
+	}
+	cmd.Flags().BoolVar(&list, "list", false, "Displays a listing by PR")
+
+	return cmd
+}
+
+func run(c *cobra.Command, _ []string) {
+	logger := logging.NewLogger(c)
+
+	readCampaignActivity := logger.StartActivity("Reading campaign data")
+	dir, err := campaign.OpenCampaign()
+	if err != nil {
+		readCampaignActivity.EndWithFailure(err)
+		return
+	}
+	readCampaignActivity.EndWithSuccess()
+
+	skippedCount := 0
+	noPrCount := 0
+
+	openCount := 0
+	mergedCount := 0
+	closedCount := 0
+
+	reactions := make(map[string]int)
+
+	detailsTable := table.New("Repository", "State", "Reviews", "URL")
+	detailsTable.WithHeaderFormatter(color.New(color.Underline).SprintfFunc())
+	detailsTable.WithFirstColumnFormatter(color.New(color.FgCyan).SprintfFunc())
+	detailsTable.WithWriter(logger.Writer())
+
+	for _, repo := range dir.Repos {
+		repoDirPath := path.Join("work", repo.OrgName, repo.RepoName) // i.e. work/org/repo
+
+		checkStatusActivity := logger.StartActivity("Checking PR status for %s", repo.FullRepoName)
+
+		// skip if the working copy does not exist
+		if _, err = os.Stat(repoDirPath); os.IsNotExist(err) {
+			checkStatusActivity.EndWithWarningf("Directory %s does not exist - has it been cloned?", repoDirPath)
+			skippedCount++
+			continue
+		}
+
+		status, err := gh.GetPrStatus(checkStatusActivity.Writer(), repoDirPath)
+		if err != nil {
+			checkStatusActivity.EndWithFailuref("Unable to fetch status: %v", err)
+			noPrCount++
+			continue
+		}
+
+		switch status.State {
+		case "OPEN":
+			openCount++
+		case "MERGED":
+			mergedCount++
+		case "CLOSED":
+			closedCount++
+		}
+
+		for _, reaction := range status.ReactionGroups {
+			reactions[reaction.Content] += reaction.Users.TotalCount
+		}
+
+		detailsTable.AddRow(repo.FullRepoName, status.State, status.ReviewDecision, status.Url)
+
+		checkStatusActivity.EndWithSuccess()
+	}
+
+	logger.Successf("turbolift pr-status completed\n")
+
+	logger.Println()
+
+	if list {
+		detailsTable.Print()
+		logger.Println()
+	}
+
+	summaryTable := table.New("State", "Count")
+	summaryTable.WithHeaderFormatter(color.New(color.Underline).SprintfFunc())
+	summaryTable.WithFirstColumnFormatter(color.New(color.FgCyan).SprintfFunc())
+	summaryTable.WithWriter(logger.Writer())
+
+	summaryTable.AddRow("Merged", mergedCount)
+	summaryTable.AddRow("Open", openCount)
+	summaryTable.AddRow("Closed", closedCount)
+	summaryTable.AddRow("Skipped", skippedCount)
+	summaryTable.AddRow("Not Found", noPrCount)
+
+	summaryTable.Print()
+
+	logger.Println()
+
+	reactionsTable := table.New("Reaction", "Count")
```

Comment:
I think this is a terrific idea to add, but perhaps behind an option? `--reactions`?

Reply:
Alternatively, perhaps not printing it if there are no reactions?